### DISCUSSION TITLE
Apply Theme to limited controls

### DISCUSSION THREAD

**Discussion by yprism** [lepoco/wpfui#1077] [Q&A] [answered]
Hi,

I have an application with custom UI controls, my issue with WPF UI is that it applies its style to everything rather than only to those provided by the library. Is it possible to make WPF UI not to apply its style to everything? Thanks in advance.

**Comment by pomianowski** [comment, marked as answer]
Hey @yprism, we currently have a global theme for all controls and we cannot do otherwise in the future.
